apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/CarrierClaimTransformerV2Test.java

```diff
@@ -506,7 +506,10 @@ public void shouldHaveDiagnosesMembers() {
             // Order doesn't matter
             diag5.getSequence(),
             new Coding("http://hl7.org/fhir/sid/icd-10", "H12345", null),
-            new Coding(null, null, null),
+            new Coding(
+                "http://hl7.org/fhir/us/carin-bb/CodeSystem/C4BBClaimDiagnosisType",
+                "secondary",
```

Comment:
chris, hate to bug you on this, but....
if you go to http://hl7.org/fhir/us/carin-bb/CodeSystem/C4BBClaimDiagnosisType, you'll see that the code == "secondary" but the display == "Secondary"...

Reply:
Gotcha, good catch